lib/workers/global/index.ts

```diff
@@ -83,9 +63,9 @@ export async function start(): Promise<0 | 1> {
         repoConfig.hostRules = [];
       }
       await repositoryWorker.renovateRepository(repoConfig);
+      setMeta({});
     }
-    setMeta({});
-    globalCache.cleanup(config);
+    globalFinalize(config);
```

Comment:
I think we should move `globalFinalize` to the final block, otherwise cache can be left over